source/subs_functions/subs_functions.bas

```diff
@@ -1325,11 +1325,11 @@ clearid
 id.n = qb64prefix$ + "LoadFont": id.Dependency = DEPENDENCY_LOADFONT
 id.subfunc = 1
 id.callname = "func__loadfont"
-id.args = 3
-id.arg = MKL$(STRINGTYPE - ISPOINTER) + MKL$(DOUBLETYPE - ISPOINTER) + MKL$(STRINGTYPE - ISPOINTER)
-id.specialformat = "?,?[,?]"
+id.args = 4
+id.arg = MKL$(STRINGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(STRINGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER)
+id.specialformat = "?,?[,[?][,[?]]]"

```

Comment:
Genuinely not sure it matters but I think it should be `[,[?][,?]]` so that you can't omit the last parameter but include the comma. In practice I don't remember how my code works that well, I'm not sure it would let you do that anyway or if it actually makes a distinction between these syntaxes.

Reply:
I don't think it matters.  Usually you can use a comma without having to include the optional parameter after it.

_PutImage , ,     '<-- This works just fine, for example, even though it's completely blank and those commas pass nothing

Reply:
The last two parameters are optional. Hence the syntax. Seems to be working for me and I tried lot of permutations and combinations. 😊